spot-ml/install_ml.sh

```diff
@@ -1,12 +0,0 @@
- #!/bin/bash
```

Comment:
We can have it. I think that can go as part of a new User Story to have an installation script. This change was more like _cleaning_ the unused code or out of date code.